portal/models/tou.py

```diff
@@ -10,9 +15,18 @@ class ToU(db.Model):
                               server_default='predates agreement_url',
                               nullable=False)
     audit_id = db.Column(db.ForeignKey('audit.id'), nullable=False)
+    type = db.Column('type', tou_types, nullable=False)
 
     audit = db.relationship('Audit', cascade="save-update", lazy='joined')
     """tracks when and by whom the terms were agreed to"""
 
     def __str__(self):
         return "ToU ({0.audit}) {0.agreement_url}".format(self)
+
+    def as_json(self):
+        d = {}
+        d['id'] = self.id
+        d['agreement_url'] = self.agreement_url
+        d['type'] = self.type
```

Comment:
Can we include a field for when it was `accepted`? use the value from the audit.timestamp